src/convert_json.cpp

```diff
@@ -17,56 +17,10 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 */
 
-#include <vector>
 #include <iostream>
 #include <sstream>
 
 #include "convert_json.h"
-#include "content/mods.h"
-#include "config.h"
-#include "log.h"
-#include "settings.h"
-#include "httpfetch.h"
-#include "porting.h"
```

Comment:
```
src/convert_json.cpp: In function ‘void fastWriteJson(const Json::Value&, std::ostream&)’:
src/convert_json.cpp:29:7: error: ‘unique_ptr’ is not a member of ‘std’
   29 |  std::unique_ptr<Json::StreamWriter> writer(builder.newStreamWriter());
      |       ^~~~~~~~~~
src/convert_json.cpp:24:1: note: ‘std::unique_ptr’ is defined in header ‘<memory>’; did you forget to ‘#include <memory>’?
   23 | #include "convert_json.h"
  +++ |+#include <memory>
   24 | 
src/convert_json.cpp:29:36: error: expected primary-expression before ‘>’ token
   29 |  std::unique_ptr<Json::StreamWriter> writer(builder.newStreamWriter());
      |                                    ^
src/convert_json.cpp:29:38: error: ‘writer’ was not declared in this scope; did you mean ‘fwrite’?
   29 |  std::unique_ptr<Json::StreamWriter> writer(builder.newStreamWriter());
      |                                      ^~~~~~
      |                                      fwrite
```

Reply:
Fixed. Weird that it worked locally and in CI.